pkg/controller/keycloak/keycloak_reconciler_test.go

```diff
@@ -581,6 +583,7 @@ func TestKeycloakReconciler_Test_Setting_Resources(t *testing.T) {
 	stateManager.SetState(common.GetStateFieldName(ControllerName, monitoringv1.ServiceMonitorsKind), true)
 	stateManager.SetState(common.GetStateFieldName(ControllerName, grafanav1alpha1.GrafanaDashboardKind), true)
 	stateManager.SetState(common.RouteKind, true)
+	stateManager.SetState(common.OpenShiftAPIServerKind, false)
```

Comment:
This one is probably not necessary. The code should assume it's `false` if it's not set.

Reply:
Agreed. Removed the same.